src/tab.group.ts

```diff
@@ -191,7 +190,9 @@ export default class GlideCoreTabGroup extends LitElement {
       // a Tab Panel.
       this.#tabElements.includes(clickedTab)
     ) {
-      this.#showTab(clickedTab);
+      this.#selectedTab = clickedTab;
```

Comment:
I wonder... as another option, could we get rid of these handlers altogether and have the individual Tab elements handle their own click and keydown events? Those events would set their `selected` states and trigger the `#onTabSelected` here in the tab group. Then that handler would just need to orchestrate the other changes to child elements

Reply:
If okay with you, I'd like to minimize the changes to resolve this bug. I put it in the PR description, but we do plan on coming back to Tabs here soon after some of your work wraps up.  I'd like us to consider something like this during that time, but get this resolved for the time being.